### DISCUSSION TITLE
Next JS 13 is now Server Component and React Hook Form is client component

### DISCUSSION THREAD

**Discussion by ugenegrg** [react-hook-form/react-hook-form#9803] [Q&A] [answered]
Hi Guys, I am doing Next JS project for with Supabase.
I have finished editing a simple table data.
But I think there might some good way of editing data with React Hook form, now that Next has made all pages server side unless we use "use client".
Can anyone send a Github link or sample link to edit the form (with pre-filled fetched data) with react hook form in Typescript.

Tools:
Next JS : 13.1.1
React Hook Form: 7.41.5

**Comment by bluebill1049** [comment, marked as answer]
I think we are limited to 'use client': https://twitter.com/HookForm/status/1586932087688658946
